odoo/fields.py

```diff
@@ -2376,6 +2376,32 @@ def __init__(self, comodel_name=Default, relation=Default, column1=Default,
             **kwargs
         )
 
+    def _get_mro_has_field_relation(self, model, name):
+        for mro in type(model).__mro__:
+            if hasattr(mro, '_abstract') and\
```

Comment:
why check _abstract? doesn't make sense

Reply:
Because abstract models have no __dict__ instantiated

Reply:
but what if the many2many field is declared in an abstract model like <code>mail.thread</code> and you search for that field in an inherited model?

Reply:
In that case, it will work (if the field is intented to be stored) because of the automatic relation property computation.

Reply:
Ok. then should be
```python
if not getattr(mro, '_abstract', False) and name in mro.__dict__:
```
because you should include the cases where there isn't <code>_abstract</code> attribute